presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -243,10 +243,19 @@ private RowId getRowId(int position)
 
     private static Path createPath(AcidInfo acidInfo, AcidInfo.DeleteDeltaInfo deleteDeltaInfo, String fileName)
     {
-        Path directory = new Path(acidInfo.getPartitionLocation(), deleteDeltaSubdir(
-                deleteDeltaInfo.getMinWriteId(),
-                deleteDeltaInfo.getMaxWriteId(),
-                deleteDeltaInfo.getStatementId()));
+        String deleteDeltaSubdir;
+        if (deleteDeltaInfo.getStatementId().isPresent()) {
+            deleteDeltaSubdir = deleteDeltaSubdir(
+                    deleteDeltaInfo.getMinWriteId(),
+                    deleteDeltaInfo.getMaxWriteId(),
+                    deleteDeltaInfo.getStatementId().getAsInt());
+        }
+        else {
+            deleteDeltaSubdir = deleteDeltaSubdir(
+                    deleteDeltaInfo.getMinWriteId(),
+                    deleteDeltaInfo.getMaxWriteId());
+        }
```

Comment:
No need to wrap lines, they are not too long. This will also make the difference between `if` branches more apparent

```
if (deleteDeltaInfo.getStatementId().isPresent()) {
        deleteDeltaSubdir = deleteDeltaSubdir(deleteDeltaInfo.getMinWriteId(), deleteDeltaInfo.getMaxWriteId(), deleteDeltaInfo.getStatementId().getAsInt());
    }
    else {
        deleteDeltaSubdir = deleteDeltaSubdir(deleteDeltaInfo.getMinWriteId(), deleteDeltaInfo.getMaxWriteId());
    }
```

Reply:
fixed.